### ISSUE TITLE
Build issue on Julia 1.

### ISSUE THREAD

**Issue by morrisonlevi** [JuliaParallel/MPI.jl#236]
I can't build MPI on Julia 1.1:

```
  Building MPI → `~/.julia/dev/MPI/deps/build.log`
┌ Error: Error building `MPI`: 
│ [ Info: Attempting to create directory /zhome/levijm/.julia/dev/MPI/deps/build
│ [ Info: Changing directory to /zhome/levijm/.julia/dev/MPI/deps/build
│ ERROR: LoadError: IOError: could not spawn `cmake -DCMAKE_INSTALL_PREFIX=/zhome/levijm/.julia/dev/MPI/deps/src -DCMAKE_LIB_INSTALL_PREFIX=/zhome/levijm/.julia/dev/MPI/deps/usr/lib ..`: no such file or directory (ENOENT)
│ Stacktrace:
│  [1] _spawn_primitive(::String, ::Cmd, ::Array{Any,1}) at ./process.jl:400
│  [2] setup_stdios(::getfield(Base, Symbol("##505#506")){Cmd}, ::Array{Any,1}) at ./process.jl:413
│  [3] _spawn at ./process.jl:412 [inlined]
│  [4] #run#515(::Bool, ::Function, ::Cmd) at ./process.jl:725
│  [5] run(::Cmd) at ./process.jl:724
│  [6] run(::BinDeps.SynchronousStepCollection) at /fslhome/levijm/.julia/packages/BinDeps/ZEval/src/BinDeps.jl:521 (repeats 3 times)
│  [7] satisfy!(::BinDeps.LibraryDependency, ::Array{DataType,1}) at /fslhome/levijm/.julia/packages/BinDeps/ZEval/src/dependencies.jl:944
│  [8] satisfy!(::BinDeps.LibraryDependency) at /fslhome/levijm/.julia/packages/BinDeps/ZEval/src/dependencies.jl:922
│  [9] top-level scope at /fslhome/levijm/.julia/packages/BinDeps/ZEval/src/dependencies.jl:977
│  [10] include at ./boot.jl:326 [inlined]
│  [11] include_relative(::Module, ::String) at ./loading.jl:1038
│  [12] include(::Module, ::String) at ./sysimg.jl:29
│  [13] include(::String) at ./client.jl:403
│  [14] top-level scope at none:0
│ in expression starting at /fslhome/levijm/.julia/dev/MPI/deps/build.jl:54
└ @ Pkg.Operations /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.1/Pkg/src/Operations.jl:1075
```

This is using GCC 7.3 and OpenMPI 3.1, though it doesn't look like it probably matters here.

I see the build bot using Julia 0.7. Is this simply not compatible with Julia 1.0+?

**Comment by lcw** [comment]
It looks like you don't have [CMake](https://cmake.org/download/) installed.  Try installing it and running `]build MPI`.

**Comment by morrisonlevi** [comment]
I do have CMake installed, although I guess I didn't have it in PATH. Thanks!

Can the error be improved here, somehow? It was unclear to me which part of the command failed, so knowing it was `cmake` and not `..` or some other piece would have helped me figure it out on my own.

**Comment by lcw** [comment]
I am happy to accept pull requests that provide better error messages for the build process!

**Comment by morrisonlevi** [comment]
Would that piece of code be part of the BinDeps package?

**Comment by lcw** [comment]
It is unclear to me how to catch the error using BinDeps.

Alternatively, we could try to avoid the error all together by using [CMakeWrapper.jl](https://github.com/JuliaPackaging/CMakeWrapper.jl) which ensures that CMake is installed and provides a BinDeps.jl-compatible CMakeProcess class.

**Comment by simonbyrne** [comment]
CMake will no longer be required in the next release (fixed by #271).
